Source/Scene/Expression.js

```diff
@@ -212,7 +202,7 @@ define([
     Expression.prototype.evaluateColor = function(frameState, feature, result) {
         ScratchStorage.reset();
         var color = this._runtimeAst.evaluate(frameState, feature);
-        return Color.clone(color, result);
+        return Color.fromCartesian4(color, result);
```

Comment:
I figured `evaluateColor` should return a `Color` object. `evaluate` will just return a `Cartesian4`.